docs/requirements.txt

```diff
@@ -1,5 +1,4 @@
 Sphinx
-sphinx_rtd_theme>1
-docutils<0.19
+furo
```

Comment:
If you pin docutils in `versions.cfg`, should you also pin it here? Same for Sphinx.